src/server/getters/get-fork-migration-totals.ts

```diff
@@ -0,0 +1,37 @@
+import * as Knex from "knex";
+import Augur from "augur.js";
+import { Address } from "../../types";
+import { ForkMigrationTotalsRow, UIForkMigrationTotals } from "../../types";
+import { normalizePayouts } from "./database";
```

Comment:
Does this actually get used anywhere? I think this can probably be removed.

Reply:
Yup! will do